AppCenterData/AppCenterDataTests/MSDataOperationProxyTests.m

```diff
@@ -392,6 +392,59 @@ - (void)testReadOperationFailsWhenPendingDelete {
                                }];
 }
 
+- (void)testLocalReadWhenCachedDocumentMaintainPendingOperation {
+
+  // If
+  XCTestExpectation *expectation = [self expectationWithDescription:@"Completed with local read."];
+  __block MSDocumentWrapper *cachedDocumentWrapper = [[MSDocumentWrapper alloc] initWithDeserializedValue:[MSDictionaryDocument alloc]
+                                                                                                jsonValue:@""
+                                                                                                partition:@"partition"
+                                                                                               documentId:@"documentId"
+                                                                                                     eTag:@""
+                                                                                          lastUpdatedDate:nil
+                                                                                         pendingOperation:kMSPendingOperationReplace
+                                                                                                    error:nil
+                                                                                          fromDeviceCache:YES];
+  __block MSDocumentWrapper *wrapper;
+  OCMStub([self.documentStoreMock readWithToken:OCMOCK_ANY documentId:OCMOCK_ANY documentType:OCMOCK_ANY]).andReturn(cachedDocumentWrapper);
```

Comment:
nit: It would be better to minimize stub scope to specific parameter values. Not a blocker.